guide/interactions/registering-slash-commands.md

```diff
@@ -0,0 +1,125 @@
+	console.log(command);
+});
+```
+
+Excellent! Now you've learned how to register guild specific application commands. We're not done yet, there's still a few more topics to cover, keep reading!
+
+
+## Bulk-update commands
+
+If you, for example, deploy your application commands when starting your application, you may want to update all commands and their changes at once. You can do this by passing an array of `ApplicationCommandData` objects to the `set()` method on either of the managers you've been introduced to above: 
+
+::: danger
+This will overwrite all existing commands on the application or guild with the new data you provided!
+:::
+
+```js {2-11,13-14}
+client.once('ready', async () => {
+	const data = [
+		{
+			name: 'ping',
+			description: 'Replies with Pong!',
+		},
+		{
+			name: 'pong',
+			description: 'Replies with Ping!',
+		},
+	];
+
+	const commands = await client.application?.commands.set(data);
+	console.log(commands);
+});
+```
+
+Perfect! You have now learned how to bulk-update application commands. 
+
+
+## Options
+
+Application commands can have `options`, think of these options like arguments to a function. You can specify them as seen below:
+
+```js {5-10}
+client.once('ready', async () => {
+	const data = {
+		name: 'echo',
+		description: 'Replies with your input!',
+		options: [{
+			name: 'input',
+			type: 'STRING',
+			description: 'The input which should be echoed back',
+			required: true,
+		}],
+	};
+
+	const command = await client.application?.commands.create(data);
+	console.log(command);
+});
+```
+
+Notice how we specified `required: true` within the options object? Doing this will make the option a required field and will prevent the user from sending the command without specifying a value for this option!
+
+Now you've learned how to create an application command with `options`. Keep reading, we have one more section to cover!
+
+## Option types
+
+As you've seen in the example of the last section we can specify the `type` of an `ApplicationCommandOption`. Listed below you'll find a list of all possible values you can pass as `ApplicationCommandOptionType`:
+
+::: tip
+For detailed explanations on the `SUB_COMMAND` and `SUB_COMMAND_GROUP` option types please see [this section](https://discord.com/developers/docs/interactions/slash-commands#subcommands-and-subcommand-groups) of the Discord API documentation.
+:::
+
+* `SUB_COMMAND` sets the option to be a sub-command
+* `SUB_COMMAND_GROUP` sets the option to be a sub-command-group
+* `STRING` sets the option to require a string value
+* `INTEGER` sets the option to require an integer value
+* `BOOLEAN` sets the option to require a boolean value
+* `USER` sets the option to require a user or snowflake as value
+* `CHANNEL` sets the option to require a channel or snowflake as value
+* `ROLE` sets the option to require a role or snowflake as value
+* `MENTIONABLE` sets the option to require a user, role or snowflake as value
```

Comment:
I believe it would make more sense to have this tip show up after the choice types because it introduces an explanation to 2 of them so it wouldn't make sense to suggest that before they're introduced in the guide.
```suggestion
* `SUB_COMMAND` sets the option to be a sub-command
* `SUB_COMMAND_GROUP` sets the option to be a sub-command-group
* `STRING` sets the option to require a string value
* `INTEGER` sets the option to require an integer value
* `BOOLEAN` sets the option to require a boolean value
* `USER` sets the option to require a user or snowflake as value
* `CHANNEL` sets the option to require a channel or snowflake as value
* `ROLE` sets the option to require a role or snowflake as value
* `MENTIONABLE` sets the option to require a user, role or snowflake as value
::: tip
For detailed explanations on the `SUB_COMMAND` and `SUB_COMMAND_GROUP` option types please see [this section](https://discord.com/developers/docs/interactions/slash-commands#subcommands-and-subcommand-groups) of the Discord API documentation.
:::
```

Reply:
Does not feel like a good idea, considering that the listing of option types will increase over time, the tip will just go under.